### PR TITLE
Hyperparameters - Updated with changes

### PR DESCRIPTION
**Pull request by neilmicallef**

### CODE REVIEW

File: terms/hyperparameters_nm.bib [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+@Article{MacKay1999,
+  author   = {MacKay, David J. C.},
+  title    = {Comparison of Approximate Methods for Handling Hyperparameters},
+  journal  = {Neural Computation},
+  year     = {1999},
+  volume   = {11},
+  number   = {5},
+  pages    = {1035-1068},
+  eprint   = {https://doi.org/10.1162/089976699300016331},
```
Comment (jp-um) [inline terms/hyperparameters_nm.bib]: Please avoid eprints and urls in article entries...

**Comment by neilmicallef** [comment]
Bibliography has been amended as requested.

**Comment by jp-um** [comment]
Your filenames in the tex file do not match your filenames in the commit.  Grid, random, Big_C.PNG.  No doubt this is because you are using windows (filenames are case insensitive).  You should test this on Ubuntu as instructed.  You are an ML practitioner, Ditch windows.

Fix filenames and get back to me.  It is probably a good idea to stick to lowercase.

**Comment by neilmicallef** [comment]
Graphics files & references fixed to lowercase. 

Also modified .bib file as there were some warnings.

**Comment by jp-um** [comment]
Still had to fix your actual filenames.  Merged.

**Comment by neilmicallef** [comment]
That's odd... I even uploaded the files individually with renamed filenames to make sure. It might be a File Explorer issue after all since it's Windows :) 

Thanks for merging!

**Comment by jp-um** [comment]
You are a computational scientist reading an advanced degree in the area.
You should i) drop windows and ii) use git from the command line.

*Dr Jean-Paul Ebejer**,*  BSc (Hons) (Melita), MSc (Imperial), DPhil (Oxon.)

*Lecturer*
Centre for Molecular Medicine and BioBanking,
Office 320, Biomedical Sciences Building,
University of Malta, Msida MSD 2080.
T: (00356) 2340 3263

<https://twitter.com/malteseunderdog>

**Pull request merged**